conda-recipe/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: csxtools
+  version: {{ environ.get('GIT_DESCRIBE_TAG', 'GIT_STUB') }}.post{{ environ.get('GIT_DESCRIBE_NUMBER','') }}
```

Comment:
'v' with the exception of a bad tag for 0.0.3.

https://github.com/scikit-beam/scikit-beam/tags